### PR TITLE
Update apt packages before installing

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Comment by kashifest** [comment]
Current fix is atleast progressing further and now failing for IPA build size.

File: ci/scripts/image_scripts/build_ipa.sh [hunk 1]
```diff
@@ -65,6 +65,8 @@ STAGING="${STAGING:-false}"
 METADATA_PATH="/tmp/metadata.txt"
 
 sudo rm -rf "${IPA_BUILD_WORKSPACE}"
+# Update apt packages
+sudo apt-get update -y
```
Comment (tuminoid) [inline ci/scripts/image_scripts/build_ipa.sh:69]: Maybe also add `apt-get clean` after all the updates and installs.

Comment (kashifest) [reply ci/scripts/image_scripts/build_ipa.sh:69]: This is done on the test VM and it should not add to the size of the IPA itself , I can add it though

Comment (kashifest) [reply ci/scripts/image_scripts/build_ipa.sh:69]: Done

**Review by smoshiur1237 — APPROVED** [review]

**Comment by kashifest** [comment]
> Are the cloned sources also cleaned up somewhere?

Not in this script atleast

**Comment by kashifest** [comment]
/override test-metal3-fullstack 
Size error issue is irrelevant to this PR

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Review by Sunnatillo — APPROVED** [review]

**Pull request merged**